arch/arm64/boot/dts/somc/kona-edo-common.dtsi

```diff
@@ -226,26 +226,32 @@
 		};
 	};
 
+	fpc_vana_vreg: fpc_vana_vreg {
+		compatible = "regulator-fixed";
+		regulator-name = "fpc_vana_fixed_reg";
+		regulator-min-microvolt = <3350000>;
+		regulator-max-microvolt = <3350000>;
+		startup-delay-us = <2000>;
+		enable-active-high;
+		gpio = <&pm8150l_gpios 8 0>;
+		vin-supply = <&pm8150a_bob>;
+	};
+
 	et6xx: et6xx {
-		status = "ok";
 		compatible = "egistec,et580";
 
-		egistec,gpio_irq = <&tlmm 23 0x0>;
-
-		pinctrl-names = "et6xx_reset_reset",
-				"et6xx_reset_active",
-				"et6xx_ldo_low",
-				"et6xx_ldo_high",
-				"et6xx_irq_active";
+		et51x,gpio_irq = <&tlmm 23 0x0>;
+		et51x,no-low-voltage-probe;
+		et51x,vana-voltage = <3350000>;
```

Comment:
Regulator-fixed, is this just to prevent errors when the driver tries to set it to 3v3 flat? Gross.

Reply:
If you have another cleaner alternative that doesn't break the whole driver logic and without hardcoding voltages again inside, you're welcome.